Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt

```diff
@@ -165,13 +167,32 @@ private fun Body(
                         modifier = HzPadding
                     )
                     Spacer(Modifier.height(16.dp))
+                    val seeMore = remember { mutableStateOf(true) }
                     Text(
                         text = stringResource(R.string.detail_placeholder),
                         style = MaterialTheme.typography.body1,
                         color = JetsnackTheme.colors.textHelp,
+                        maxLines = if (seeMore.value) 5 else Int.MAX_VALUE,
+                        overflow = TextOverflow.Ellipsis,
                         modifier = HzPadding
                     )
-
+                    val textButton = if (seeMore.value)
```

Comment:
More meaningful name please

Reply:
Also can we have curly braces around the clauses please (or inline the `if` within the `stringResource`?)